src/sqlfluff/dialects/dialect_databricks.py

```diff
@@ -648,6 +648,30 @@ class CreateDatabaseStatementSegment(sparksql.CreateDatabaseStatementSegment):
     )
 
 
+class CreateViewStatementSegment(sparksql.CreateViewStatementSegment):
+    """A `CREATE VIEW` statement.
+
+    https://docs.databricks.com/aws/en/sql/language-manual/sql-ref-syntax-ddl-create-view
+    https://docs.databricks.com/aws/en/dlt-ref/dlt-sql-ref-create-materialized-view
+    """
+
+    match_grammar = sparksql.CreateViewStatementSegment.match_grammar.copy(
+        insert=[
+            OneOf(
+                Sequence(
+                    Ref.keyword("PRIVATE", optional=True),
+                    Ref.keyword("MATERIALIZED"),
+                ),
+                Ref.keyword("MATERIALIZED", optional=True),
+            ),
```

Comment:
This appears to break the base `CREATE VIEW`.

This now fails:
```sql
CREATE VIEW view_name AS SELECT 1;
```

```suggestion
            Sequence(
                Ref.keyword("PRIVATE", optional=True),
                Ref.keyword("MATERIALIZED"),
                optional=True,
            ),
```